ldk/javascript/src/network/index.ts

```diff
@@ -20,9 +22,7 @@ export interface HTTPResponse {
   /**
    * The HTTP response as a byte array. To decode into a UTF-8 string you can:
    * ```
-   * import * as util from 'util';
-   * ...
-   * var string = new TextDecoder("utf-8").decode(uint8array);
+   let decodedText = network.decode(data);
```

Comment:
I'm wondering whether we should accept multiple types for the body and handle the conversion internally.

So if we say that `data` takes `Uint8Array|string`, we can see if the `data` property is a `string` and if so, encode it as UTF-8 for the user.

Reply:
Yes, that makes sense. I could create a card to prioritise.